openupgrade_scripts/apriori.py

```diff
@@ -0,0 +1,25 @@
+""" Encode any known changes to the database here
+to help the matching process
+"""
+
+# Renamed modules is a mapping from old module name to new module name
+renamed_modules = {
+    # odoo
+    # OCA/...
+}
+
+# Merged modules contain a mapping from old module names to other,
+# preexisting module names
+merged_modules = {
+    # odoo
+    # OCA/...
+}
+
+# only used here for upgrade_analysis
+renamed_models = {
```

Comment:
The analysis has to be re-run if renamed models are found. Do you know if there's any right now?

Reply:
No, not aware of any.

Reply:
The re-run will have to be done few months later, anyway.